projects/cm_mcu/I2CCommunication.c

```diff
@@ -39,9 +39,9 @@
 // TivaWare includes
 #include "driverlib/uart.h"
 
-//#include "MonitorTask.h"
```

Comment:
pls delete

Reply:
you mean deleting `#include "driverlib/uart.h"` ?

Reply:
please disregard, I think I misread the diff. I thought the header had been commented out but not removed.